.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
do we actually want to commit this? Or just keep locally?

Reply:
I don't have a strong opinion. I see three options:
1. Commit it
2. Don't commit it
3. Move it to `.vscode/settings.json.default` or some other file so others have a reference of it.

@vdmgolub might have input as well.

Reply:
If I understand the issue origin correctly, I would fix the trailing spaces issue, I don't think it's a good thing to have golden files with them. This setting will allow more trailing spaces in future, which will make the issue worse. So I would vote for 2 as eventually redundant.